src/uct/api/uct.h

```diff
@@ -2556,7 +2556,8 @@ UCT_INLINE_API ucs_status_t uct_ep_atomic64_fetch(uct_ep_h ep, uct_atomic_op_t o
  *                    the "func" field.
  *                    After passed to the function, the request is owned by UCT,
  *                    until the callback is called and returns UCS_OK.
- * @param [in]  flags Reserved for future use.
+ * @param [in]  flags Flags that control pending request processing (currently only
```

Comment:
can you pls remote "(currently only ...)", replace by `see @ref <name-of-enum>`
even one flag counts..

Reply:
done